### DISCUSSION TITLE
Injecting mediamtx with path io configurations file

### DISCUSSION THREAD

**Discussion by RobotnickIsrael** [bluenviron/mediamtx#3046] [Questions]
### Question

I'm running the mediamtx binary, but I want the configuration file in my git project source code and the binary outside, meaning they should sit in different folders.

Can I give the path to the configuration file as a parameter when I run it? 
Couldn't find this option on the docs

*Using mediamtx 1.5.1 binary for Linux.

**Comment by aler9** [comment]
Hello, yes, you can do it by passing the path to the configuration file as first argument:

```
./mediamtx my_conf_file.yml
```

[a comment by an automated account is not shown]
